hyperparams/tqc.yml

```diff
@@ -165,7 +165,6 @@ Swimmer-v3:
 
 # === HER Robotics GoalEnvs ===
 FetchReach-v1:
-  env_wrapper: sb3_contrib.common.wrappers.TimeFeatureWrapper
```

Comment:
@araffin can you explain why you remove the TimeFeatureWrapper?

Reply:
before, the timeout should have been handled properly, but after quick runs, it seemed that the wrapper make the performance more consistent (my jugement was probably false back then, i think the real issue was the one discussed with AdamW).
With the new implementation, they should be properly handled now.

Reply:
which mean no need for the wrapper.